sm_cnn/requirements.txt

```diff
@@ -1,5 +1,4 @@
-gensim==1.0.1
-nltk==3.2.1
-numpy==1.11.3
-pandas==0.19.2
-pytorch==0.1.12
+nltk==3.2.4
+numpy==1.13.1
+pytorch==0.2.0+5f864ca
```

Comment:
please remove the commit number from the end

Reply:
@tuzhucheng I think the tutorial and `nn-architecture.png` are already deleted, right?